AWS AutoScalingTerminationStateServiceTest regularly fails

The `AutoScalingTerminationStateServiceTest.testLifeCyclePolling` fails on an irregular basis making both buildbot and [Travis](https://api.travis-ci.org/v3/job/592679363/log.txt) fail from time to time:

```
[INFO] --- maven-surefire-plugin:3.0.0-M3:test (default-test) @ opencast-termination-state-aws ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.583 s <<< FAILURE! - in org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
[ERROR] testLifeCyclePolling(org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest)  Time elapsed: 3.44 s  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest.testLifeCyclePolling(AutoScalingTerminationStateServiceTest.java:132)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   AutoScalingTerminationStateServiceTest.testLifeCyclePolling:132 expected:<1> but was:<0>
[INFO] 
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

Looking at the high number of today's pull requests you could see this error popping up multiple times, breaking a number of test builds. @gregorydlogan, assuming that you wrote those tests, can you take a look?

<details>
<summary>Mudule build log of error copied from buildbot</summary>

```
[INFO] Building Opencast :: termination-state-aws 9-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ opencast-termination-state-aws ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-maven) @ opencast-termination-state-aws ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ opencast-termination-state-aws ---
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/builds/Pull_Request_Build/build/modules/termination-state-aws' && 'git' 'rev-parse' '--verify' '--short=7' 'HEAD'
[INFO] Working directory: /builds/Pull_Request_Build/build/modules/termination-state-aws
[INFO] Storing buildNumber: bb7a0ea at timestamp: 1572520502642
[INFO] Storing buildScmBranch: refs/pull/1184/merge
[INFO] 
[INFO] --- maven-checkstyle-plugin:3.1.0:check (default) @ opencast-termination-state-aws ---
[INFO] Starting audit...
Audit done.
[INFO] 
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ opencast-termination-state-aws ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ opencast-termination-state-aws ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /builds/Pull_Request_Build/build/modules/termination-state-aws/target/classes
[INFO] /builds/Pull_Request_Build/build/modules/termination-state-aws/src/main/java/org/opencastproject/terminationstate/endpoint/aws/AutoScalingTerminationStateRestService.java: /builds/Pull_Request_Build/build/modules/termination-state-aws/src/main/java/org/opencastproject/terminationstate/endpoint/aws/AutoScalingTerminationStateRestService.java uses unchecked or unsafe operations.
[INFO] /builds/Pull_Request_Build/build/modules/termination-state-aws/src/main/java/org/opencastproject/terminationstate/endpoint/aws/AutoScalingTerminationStateRestService.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-resources-plugin:3.0.2:testResources (default-testResources) @ opencast-termination-state-aws ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] skip non existing resourceDirectory /builds/Pull_Request_Build/build/modules/termination-state-aws/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:testCompile (default-testCompile) @ opencast-termination-state-aws ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /builds/Pull_Request_Build/build/modules/termination-state-aws/target/test-classes
[INFO] /builds/Pull_Request_Build/build/modules/termination-state-aws/src/test/java/org/opencastproject/terminationstate/aws/AutoScalingTerminationStateServiceTest.java: /builds/Pull_Request_Build/build/modules/termination-state-aws/src/test/java/org/opencastproject/terminationstate/aws/AutoScalingTerminationStateServiceTest.java uses unchecked or unsafe operations.
[INFO] /builds/Pull_Request_Build/build/modules/termination-state-aws/src/test/java/org/opencastproject/terminationstate/aws/AutoScalingTerminationStateServiceTest.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M3:test (default-test) @ opencast-termination-state-aws ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
11:15:04  INFO (AutoScalingTerminationStateService:277) - [>e90755a3] Started polling for Lifecycle state change
11:15:04 DEBUG (AutoScalingTerminationStateService$CheckLifeCycleState:309) - [>e90755a3] Lifecycle state is InService
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.574 s <<< FAILURE! - in org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
[ERROR] testLifeCyclePolling(org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest)  Time elapsed: 3.451 s  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest.testLifeCyclePolling(AutoScalingTerminationStateServiceTest.java:132)
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   AutoScalingTerminationStateServiceTest.testLifeCyclePolling:132 expected:<1> but was:<0>
[INFO] 
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] opencast ........................................... SUCCESS [  2.653 s]
[INFO] Opencast :: common ................................. SUCCESS [ 46.169 s]
[INFO] Opencast :: metadata-api ........................... SUCCESS [  2.733 s]
[INFO] Opencast :: workspace-api .......................... SUCCESS [  1.555 s]
[INFO] Opencast :: dublincore ............................. SUCCESS [  4.697 s]
[INFO] Opencast :: workflow-service-api ................... SUCCESS [  1.823 s]
[INFO] Opencast :: message-broker-api ..................... SUCCESS [  1.637 s]
[INFO] Opencast :: series-service-api ..................... SUCCESS [  1.536 s]
[INFO] Opencast :: asset-manager-api ...................... SUCCESS [  2.768 s]
[INFO] Opencast :: authorization-xacml .................... SUCCESS [  4.103 s]
[INFO] Opencast :: common-jpa-impl ........................ SUCCESS [  5.783 s]
[INFO] Opencast :: rest-test-environment .................. SUCCESS [  0.962 s]
[INFO] Opencast :: authorization-manager .................. SUCCESS [ 11.368 s]
[INFO] Opencast :: capture-admin-service-api .............. SUCCESS [  1.497 s]
[INFO] Opencast :: asset-manager-util ..................... SUCCESS [  1.496 s]
[INFO] Opencast :: search-api ............................. SUCCESS [  1.253 s]
[INFO] Opencast :: scheduler-api .......................... SUCCESS [  3.166 s]
[INFO] Opencast :: event-comment .......................... SUCCESS [  4.652 s]
[INFO] Opencast :: search ................................. SUCCESS [ 21.319 s]
[INFO] Opencast :: ingest-service-api ..................... SUCCESS [  1.449 s]
[INFO] Opencast :: urlsigning-common ...................... SUCCESS [  2.501 s]
[INFO] Opencast :: urlsigning-service-api ................. SUCCESS [  1.322 s]
[INFO] Opencast :: kernel ................................. SUCCESS [ 12.037 s]
[INFO] Opencast :: userdirectory .......................... SUCCESS [  9.579 s]
[INFO] Opencast :: search-service-api ..................... SUCCESS [  1.536 s]
[INFO] Opencast :: distribution-service-api ............... SUCCESS [  1.714 s]
[INFO] Opencast :: publication-service-api ................ SUCCESS [  1.480 s]
[INFO] Opencast :: distribution-workflowoperation ......... SUCCESS [  3.095 s]
[INFO] Opencast :: index-service .......................... SUCCESS [  7.877 s]
[INFO] Opencast :: presets ................................ SUCCESS [  2.483 s]
[INFO] Opencast :: static-file-service-api ................ SUCCESS [  2.679 s]
[INFO] Opencast :: statistics-service-api ................. SUCCESS [  6.480 s]
[INFO] Opencast :: statistics-export-service .............. SUCCESS [  3.018 s]
[INFO] Opencast :: themes ................................. SUCCESS [  3.871 s]
[INFO] Opencast :: smil-api ............................... SUCCESS [  1.488 s]
[INFO] Opencast :: composer-service-api ................... SUCCESS [  2.333 s]
[INFO] Opencast :: solr ................................... SUCCESS [  3.208 s]
[INFO] Opencast :: series-service-impl .................... SUCCESS [ 10.047 s]
[INFO] Opencast :: asset-manager-impl ..................... SUCCESS [ 33.975 s]
[INFO] Opencast :: admin-ui-index ......................... SUCCESS [  2.780 s]
[INFO] Opencast :: admin-ui ............................... SUCCESS [02:33 min]
[INFO] Opencast :: animate-api ............................ SUCCESS [  1.470 s]
[INFO] Opencast :: animate-impl ........................... SUCCESS [  8.188 s]
[INFO] Opencast :: animate-remote ......................... SUCCESS [  1.451 s]
[INFO] Opencast :: inspection-service-api ................. SUCCESS [  1.745 s]
[INFO] Opencast :: animate-workflowoperation .............. SUCCESS [  3.004 s]
[INFO] Opencast :: annotation-api ......................... SUCCESS [  1.459 s]
[INFO] Opencast :: annotation-impl ........................ SUCCESS [  4.465 s]
[INFO] Opencast :: asset-manager-workflowoperation ........ SUCCESS [  1.637 s]
[INFO] Opencast :: asset-manager-storage-aws .............. SUCCESS [  9.130 s]
[INFO] Opencast :: asset-manager-storage-fs ............... SUCCESS [  2.567 s]
[INFO] Opencast :: caption-api ............................ SUCCESS [  1.408 s]
[INFO] Opencast :: mpeg7 .................................. SUCCESS [  3.003 s]
[INFO] Opencast :: caption-impl ........................... SUCCESS [  2.811 s]
[INFO] Opencast :: caption-remote ......................... SUCCESS [  1.412 s]
[INFO] Opencast :: capture-admin-service-impl ............. SUCCESS [ 11.143 s]
[INFO] Opencast :: comments-workflowoperation ............. SUCCESS [  2.493 s]
[INFO] Opencast :: composer-ffmpeg ........................ SUCCESS [ 16.615 s]
[INFO] Opencast :: composer-service-remote ................ SUCCESS [  1.526 s]
[INFO] Opencast :: inspection-workflowoperation ........... SUCCESS [  2.938 s]
[INFO] Opencast :: composer-workflowoperation ............. SUCCESS [ 14.648 s]
[INFO] Opencast :: oaipmh-persistence ..................... SUCCESS [  5.309 s]
[INFO] Opencast :: conductor .............................. SUCCESS [  3.412 s]
[INFO] Opencast :: cover-image-api ........................ SUCCESS [  1.423 s]
[INFO] Opencast :: cover-image-impl ....................... SUCCESS [  6.594 s]
[INFO] Opencast :: cover-image-remote ..................... SUCCESS [  1.431 s]
[INFO] Opencast :: cover-image-workflowoperation .......... SUCCESS [  2.254 s]
[INFO] Opencast :: crop-api ............................... SUCCESS [  1.701 s]
[INFO] Opencast :: inspection-service-ffmpeg .............. SUCCESS [  3.388 s]
[INFO] Opencast :: crop-ffmpeg ............................ SUCCESS [  7.826 s]
[INFO] Opencast :: crop-remote ............................ SUCCESS [  1.391 s]
[INFO] Opencast :: crop-workflowoperation ................. SUCCESS [  2.600 s]
[INFO] Opencast :: db ..................................... SUCCESS [  3.207 s]
[INFO] Opencast :: dictionary-api ......................... SUCCESS [  1.383 s]
[INFO] Opencast :: dictionary-hunspell .................... SUCCESS [  2.931 s]
[INFO] Opencast :: dictionary-none ........................ SUCCESS [  2.134 s]
[INFO] Opencast :: dictionary-regexp ...................... SUCCESS [  2.249 s]
[INFO] Opencast :: distribution-service-aws-s3-api ........ SUCCESS [  1.407 s]
[INFO] Opencast :: distribution-service-aws-s3 ............ SUCCESS [  4.728 s]
[INFO] Opencast :: distribution-service-aws-s3-remote ..... SUCCESS [  1.654 s]
[INFO] Opencast :: distribution-service-download .......... SUCCESS [  5.963 s]
[INFO] Opencast :: distribution-service-download-remote ... SUCCESS [  1.950 s]
[INFO] Opencast :: distribution-service-adaptive-streaming-wowza SUCCESS [  2.644 s]
[INFO] Opencast :: distribution-service-streaming-remote .. SUCCESS [  1.452 s]
[INFO] Opencast :: email-template-service-api ............. SUCCESS [  1.395 s]
[INFO] Opencast :: email-template-service-impl ............ SUCCESS [  3.176 s]
[INFO] Opencast :: engage-paella-player ................... SUCCESS [ 35.764 s]
[INFO] Opencast :: engage-theodul-api ..................... SUCCESS [  1.318 s]
[INFO] Opencast :: engage-theodul-core .................... SUCCESS [ 12.426 s]
[INFO] Opencast :: engage-theodul-plugin-archetype ........ SUCCESS [  1.924 s]
[INFO] Opencast :: engage-theodul-plugin-controls ......... SUCCESS [  6.377 s]
[INFO] Opencast :: engage-theodul-plugin-custom-mhConnection SUCCESS [  4.203 s]
[INFO] Opencast :: engage-theodul-plugin-custom-notifications SUCCESS [  4.086 s]
[INFO] Opencast :: engage-theodul-plugin-custom-usertracking SUCCESS [  4.358 s]
[INFO] Opencast :: engage-theodul-plugin-custom-matomo .... SUCCESS [  1.370 s]
[INFO] Opencast :: engage-theodul-plugin-description ...... SUCCESS [  4.205 s]
[INFO] Opencast :: engage-theodul-plugin-tab-description .. SUCCESS [  4.623 s]
[INFO] Opencast :: engage-theodul-plugin-tab-shortcuts .... SUCCESS [  4.459 s]
[INFO] Opencast :: engage-theodul-plugin-tab-slidetext .... SUCCESS [  4.210 s]
[INFO] Opencast :: engage-theodul-plugin-timeline-statistics SUCCESS [  4.109 s]
[INFO] Opencast :: engage-theodul-plugin-video-videojs .... SUCCESS [  5.479 s]
[INFO] Opencast :: engage-ui .............................. SUCCESS [ 10.875 s]
[INFO] Opencast :: execute-api ............................ SUCCESS [  1.427 s]
[INFO] Opencast :: execute-impl ........................... SUCCESS [  3.274 s]
[INFO] Opencast :: execute-remote ......................... SUCCESS [  1.439 s]
[INFO] Opencast :: execute-workflowoperation .............. SUCCESS [  2.731 s]
[INFO] Opencast :: external-api-index ..................... SUCCESS [  1.826 s]
[INFO] Opencast :: external-api ........................... SUCCESS [ 10.076 s]
[INFO] Opencast :: fileupload ............................. SUCCESS [  2.104 s]
[INFO] Opencast :: hello-world-api ........................ SUCCESS [  1.327 s]
[INFO] Opencast :: hello-world-impl ....................... SUCCESS [  2.256 s]
[INFO] Opencast :: incident-workflowoperation ............. SUCCESS [  1.466 s]
[INFO] Opencast :: ingest-download-service-api ............ SUCCESS [  1.482 s]
[INFO] Opencast :: working-file-repository-service-api .... SUCCESS [  1.518 s]
[INFO] Opencast :: ingest-download-service-impl ........... SUCCESS [  3.254 s]
[INFO] Opencast :: ingestdownloadservice-remote ........... SUCCESS [  1.481 s]
[INFO] Opencast :: working-file-repository-service-impl ... SUCCESS [  2.914 s]
[INFO] Opencast :: ingest-service-impl .................... SUCCESS [  4.932 s]
[INFO] Opencast :: ingest-download-service-workflowoperation SUCCESS [  1.469 s]
[INFO] Opencast :: inspection-service-remote .............. SUCCESS [  1.502 s]
[INFO] Opencast :: statistics-writer-workflowoperation .... SUCCESS [  1.798 s]
[INFO] Opencast :: live-schedule-api ...................... SUCCESS [  1.490 s]
[INFO] Opencast :: live-schedule-impl ..................... SUCCESS [  3.586 s]
[INFO] Opencast :: logging-workflowoperation .............. SUCCESS [  2.794 s]
[INFO] Opencast :: lti .................................... SUCCESS [ 10.900 s]
[INFO] Opencast :: mattermost-notification-workflowoperation SUCCESS [  1.576 s]
[INFO] Opencast :: message-broker-impl .................... SUCCESS [  8.472 s]
[INFO] Opencast :: metadata ............................... SUCCESS [  1.545 s]
[INFO] Opencast :: migration .............................. SUCCESS [  5.854 s]
[INFO] Opencast :: notification-workflowoperation ......... SUCCESS [  3.734 s]
[INFO] Opencast :: oaipmh-api ............................. SUCCESS [  1.508 s]
[INFO] Opencast :: oaipmh ................................. SUCCESS [ 10.672 s]
[INFO] Opencast :: oaipmh-remote .......................... SUCCESS [  1.484 s]
[INFO] Opencast :: publication-service-configurable ....... SUCCESS [  2.330 s]
[INFO] Opencast :: publication-service-configurable-remote  SUCCESS [  1.609 s]
[INFO] Opencast :: publication-service-oaipmh ............. SUCCESS [  3.060 s]
[INFO] Opencast :: publication-service-oaipmh-remote ...... SUCCESS [  5.141 s]
[INFO] Opencast :: publication-service-youtube-remote ..... SUCCESS [  1.537 s]
[INFO] Opencast :: publication-service-youtube-v3 ......... SUCCESS [  7.896 s]
[INFO] Opencast :: runtime-info ........................... SUCCESS [  2.872 s]
[INFO] Opencast :: runtime-info-ui ........................ SUCCESS [ 10.800 s]
[INFO] Opencast :: runtime-info-ui-ng ..................... SUCCESS [ 10.423 s]
[INFO] Opencast :: scheduler-impl ......................... SUCCESS [ 34.364 s]
[INFO] Opencast :: scheduler-conflict-handler ............. SUCCESS [  2.673 s]
[INFO] Opencast :: scheduler-conflict-notifier-comment .... SUCCESS [  2.926 s]
[INFO] Opencast :: scheduler-conflict-notifier-email ...... SUCCESS [  3.679 s]
[INFO] Opencast :: scheduler-remote ....................... SUCCESS [  1.622 s]
[INFO] Opencast :: schema ................................. SUCCESS [  2.565 s]
[INFO] Opencast :: search-service-feeds ................... SUCCESS [  1.505 s]
[INFO] Opencast :: search-service-impl .................... SUCCESS [ 24.000 s]
[INFO] Opencast :: search-service-remote .................. SUCCESS [  1.451 s]
[INFO] Opencast :: security-shibboleth .................... SUCCESS [  2.210 s]
[INFO] Opencast :: security-aai ........................... SUCCESS [  1.697 s]
[INFO] Opencast :: security-cas-client-wrapper ............ SUCCESS [  1.794 s]
[INFO] Opencast :: security-cas ........................... SUCCESS [  1.380 s]
[INFO] Opencast :: security-ldap .......................... SUCCESS [  1.277 s]
[INFO] Opencast :: security-openid ........................ SUCCESS [  1.491 s]
[INFO] Opencast :: series-service-remote .................. SUCCESS [  1.949 s]
[INFO] Opencast :: serviceregistry ........................ SUCCESS [ 29.178 s]
[INFO] Opencast :: silencedetection-api ................... SUCCESS [  2.419 s]
[INFO] Opencast :: silencedetection-impl .................. SUCCESS [  2.911 s]
[INFO] Opencast :: silencedetection-remote ................ SUCCESS [  1.412 s]
[INFO] Opencast :: smil-impl .............................. SUCCESS [  2.823 s]
[INFO] Opencast :: sox-api ................................ SUCCESS [  1.782 s]
[INFO] Opencast :: sox-impl ............................... SUCCESS [  2.901 s]
[INFO] Opencast :: sox-remote ............................. SUCCESS [  1.510 s]
[INFO] Opencast :: sox-workflowoperation .................. SUCCESS [  3.024 s]
[INFO] Opencast :: static ................................. SUCCESS [  2.461 s]
[INFO] Opencast :: static-file-service-impl ............... SUCCESS [  2.574 s]
[INFO] Opencast :: statistics-service-impl ................ SUCCESS [  1.889 s]
[INFO] Opencast :: statistics-service-remote .............. SUCCESS [  1.601 s]
[INFO] Opencast :: statistics-provider-influx ............. SUCCESS [  3.237 s]
[INFO] Opencast :: statistics-provider-random ............. SUCCESS [  1.532 s]
[INFO] Opencast :: termination-state-api .................. SUCCESS [  2.550 s]
[INFO] Opencast :: termination-state-aws .................. FAILURE [  5.306 s]
[INFO] Opencast :: termination-state-impl ................. SKIPPED
[INFO] Opencast :: textanalyzer-api ....................... SKIPPED
[INFO] Opencast :: textanalyzer-impl ...................... SKIPPED
[INFO] Opencast :: textanalyzer-remote .................... SKIPPED
[INFO] Opencast :: textanalyzer-workflowoperation ......... SKIPPED
[INFO] Opencast :: textextractor-tesseract ................ SKIPPED
[INFO] Opencast :: themes-workflowoperation ............... SKIPPED
[INFO] Opencast :: timelinepreviews-api ................... SKIPPED
[INFO] Opencast :: timelinepreviews-ffmpeg ................ SKIPPED
[INFO] Opencast :: timelinepreviews-remote ................ SKIPPED
[INFO] Opencast :: timelinepreviews-workflowoperation ..... SKIPPED
[INFO] Opencast :: transcription-service-api .............. SKIPPED
[INFO] Opencast :: transcription-service-persistence ...... SKIPPED
[INFO] Opencast :: transcription-service-google-speech-impl SKIPPED
[INFO] Opencast :: transcription-service-ibm-watson-impl .. SKIPPED
[INFO] Opencast :: transcription-service-workflowoperation  SKIPPED
[INFO] Opencast :: urlsigning-service-impl ................ SKIPPED
[INFO] Opencast :: urlsigning-verifier-service-api ........ SKIPPED
[INFO] Opencast :: urlsigning-verifier-service-impl ....... SKIPPED
[INFO] Opencast :: userdirectory-ldap ..................... SKIPPED
[INFO] Opencast :: userdirectory-moodle ................... SKIPPED
[INFO] Opencast :: userdirectory-sakai .................... SKIPPED
[INFO] Opencast :: userdirectory-brightspace .............. SKIPPED
[INFO] Opencast :: user-interface-configuration ........... SKIPPED
[INFO] Opencast :: usertracking-api ....................... SKIPPED
[INFO] Opencast :: usertracking-impl ...................... SKIPPED
[INFO] Opencast :: videoeditor-api ........................ SKIPPED
[INFO] Opencast :: videoeditor-ffmpeg-impl ................ SKIPPED
[INFO] Opencast :: videoeditor-remote ..................... SKIPPED
[INFO] Opencast :: workflow-workflowoperation ............. SKIPPED
[INFO] Opencast :: videoeditor-workflowoperation .......... SKIPPED
[INFO] Opencast :: videosegmenter-api ..................... SKIPPED
[INFO] Opencast :: videosegmenter-ffmpeg .................. SKIPPED
[INFO] Opencast :: videosegmenter-remote .................. SKIPPED
[INFO] Opencast :: videosegmenter-workflowoperation ....... SKIPPED
[INFO] Opencast :: waveform-api ........................... SKIPPED
[INFO] Opencast :: waveform-ffmpeg ........................ SKIPPED
[INFO] Opencast :: waveform-remote ........................ SKIPPED
[INFO] Opencast :: waveform-workflowoperation ............. SKIPPED
[INFO] Opencast :: workflow-condition-parser .............. SKIPPED
[INFO] Opencast :: workflow-service-impl .................. SKIPPED
[INFO] Opencast :: workflow-service-remote ................ SKIPPED
[INFO] Opencast :: working-file-repository-service-remote . SKIPPED
[INFO] Opencast :: workspace-impl ......................... SKIPPED
[INFO] Assemblies ......................................... SKIPPED
[INFO] Assemblies // Apache Karaf Features ................ SKIPPED
[INFO] Assemblies // Distribution Admin ................... SKIPPED
[INFO] Assemblies // Distribution Admin/Presentation ...... SKIPPED
[INFO] Assemblies // Distribution All-In-One .............. SKIPPED
[INFO] Assemblies // Distribution Ingest .................. SKIPPED
[INFO] Assemblies // Distribution Presentation ............ SKIPPED
[INFO] Assemblies // Distribution Worker .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16:13 min
[INFO] Finished at: 2019-10-31T11:15:07+00:00
[INFO] Final Memory: 315M/910M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M3:test (default-test) on project opencast-termination-state-aws: There are test failures.
[ERROR] 
[ERROR] Please refer to /builds/Pull_Request_Build/build/modules/termination-state-aws/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :opencast-termination-state-aws
```
</details>

Okay, I have seen this again and again over time, regularly breaking builds and confusing people.
It also seems like the test was activated again:

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.508 s <<< FAILURE! - in org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest
Error:  testLifeCyclePolling(org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest)  Time elapsed: 0.344 s  <<< FAILURE!
java.lang.AssertionError: 

  Unexpected method call AmazonAutoScaling.completeLifecycleAction({LifecycleHookName: test-terminate,AutoScalingGroupName: test-auto-scaling,LifecycleActionResult: CONTINUE,}):
    AmazonAutoScaling.completeLifecycleAction(<any>): expected: 1, actual: 2
    AmazonAutoScaling.shutdown(): expected: 1, actual: 0
	at org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest.testLifeCyclePolling(AutoScalingTerminationStateServiceTest.java:151)

[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    AutoScalingTerminationStateServiceTest.testLifeCyclePolling:151 
  Unexpected method call AmazonAutoScaling.completeLifecycleAction({LifecycleHookName: test-terminate,AutoScalingGroupName: test-auto-scaling,LifecycleActionResult: CONTINUE,}):
    AmazonAutoScaling.completeLifecycleAction(<any>): expected: 1, actual: 2
    AmazonAutoScaling.shutdown(): expected: 1, actual: 0
[INFO] 
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for opencast 10-SNAPSHOT:
[INFO] 
```

I will create a pull request deactivating this again since this hurts development and testing more than it helps in its current state.

--- a/modules/termination-state-aws/src/test/java/org/opencastproject/terminationstate/aws/AutoScalingTerminationStateServiceTest.java
+++ b/modules/termination-state-aws/src/test/java/org/opencastproject/terminationstate/aws/AutoScalingTerminationStateServiceTest.java
@@ -38,6 +38,7 @@ import org.easymock.IAnswer;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.quartz.JobDetail;
 import org.quartz.JobExecutionContext;
@@ -128,6 +129,7 @@ public class AutoScalingTerminationStateServiceTest {
   }
 
   @Test
+  @Ignore // regularly fails. See https://github.com/opencast/opencast/issues/1111
   public void testLifeCyclePolling() throws Exception {
     service.startPollingLifeCycleHook();
     String[] trigger = scheduler.getTriggerNames(AutoScalingTerminationStateService.SCHEDULE_GROUP);

